ci/lint.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+
+if [[ -n "${DEBUG-}" ]]; then
+  set -x
+fi
+
+cd "$(dirname "$0")/.."
+
+./scripts/lint.sh ./tools --check
```

Comment:
why no newline at the end of here, but we keep it in `scripts/lint.sh`?

Reply:
same question for the Dockerfile

Reply:
Forgot to add them 🤷‍♂

Reply:
fair enough :P